system/lib/libc/musl/src/stdio/__stdio_seek.c

```diff
@@ -4,7 +4,13 @@ off_t __stdio_seek(FILE *f, off_t off, int whence)
 {
 	off_t ret;
 #ifdef SYS__llseek
-	if (syscall(SYS__llseek, f->fd, off>>32, off, &ret, whence)<0)
+	off_t off_hi =
+#ifndef __EMSCRIPTEN__
+		off >> 32;
```

Comment:
All other targets use `_Int64` for their `off_t` size (https://github.com/bpowers/musl/blob/94cb2ec2a0ffcb47d24dbf7a30e462505396cf54/include/alltypes.h.in#L23), we deviated from that in Emscripten's arch-specific alltypes.h (https://github.com/kripken/emscripten/blob/master/system/lib/libc/musl/arch/emscripten/bits/alltypes.h#L225)
Removing that deviation was option 3